lib/experimental/fonts-api/class-wp-fonts-resolver.php

```diff
@@ -219,7 +214,12 @@ private static function get_settings() {
 			);
 
 			// Make sure there are no duplicates.
-			$settings['typography']['fontFamilies'] = array_unique( $settings['typography']['fontFamilies'] );
+			$settings['typography']['fontFamilies'] = array_unique( $settings['typography']['fontFamilies'], SORT_REGULAR );
```

Comment:
This doesn't make sense to be in the loop.

Reply:
This is the existing behavior for a long time, I think it's out of the scope of this PR to address the fatal error. We can plan to move it outside the loop in the follow-up PR. Does it make sense?

Reply:
> I'm not sure why we want to remove the duplicates on $settings['typography']['fontFamilies']. Instead, I think doing it on $settings['typography']['fontFamilies']['theme']

Yep, same. Thinking this is one of the problems here that caused the fatal error (see comment below). I'd be a +1 to fix that `array_unique()` as it seems to be a bug.